packages/repository/src/decorators/repository.ts

```diff
@@ -24,15 +25,15 @@ export class RepositoryMetadata {
   /**
    * Class of the model
    */
-  modelClass?: Class<Model>;
+  modelClass?: typeof juggler.PersistedModel | typeof Model;
```

Comment:
Nice. Are there any other places where `Class<T>` can be replaced with `typeof T`? Can we perhaps get rid of `Class<>` entirely?